app/uk/gov/hmrc/partnershipidentificationfrontend/service/AuditService.scala

```diff
@@ -50,9 +51,17 @@ class AuditService @Inject()(auditConnector: AuditConnector,
   private def auditType(partnershipType: PartnershipType): String = partnershipType match {
     case GeneralPartnership => "GeneralPartnershipEntityRegistration"
     case ScottishPartnership => "ScottishPartnershipEntityRegistration"
+    case LimitedPartnership => "LimitedPartnershipRegistration"
     case _ => ""
   }
 
+  private def companyNumberJson(optCompanyProfile: Option[CompanyProfile]): JsObject = optCompanyProfile.map(_.companyNumber) match {
+    case Some(companyNumber) => Json.obj(
+      "companyNumber" -> companyNumber
+    )
+    case None => Json.obj()
+  }
+
```

Comment:
```suggestion
  private def companyNumberJson(optCompanyProfile: Option[CompanyProfile]): JsObject =
    optCompanyProfile match {
      case Some(companyProfile) => Json.obj(
        "companyNumber" -> companyProfile.companyNumber
      )
    case None => Json.obj()
  }
```

A little more concise.

Reply:
Perfect ... I willl update it